### PR TITLE
Added to-do manual in README for Raspicam library

### PR DESCRIPTION
**Pull request by Captain-DarkO**

### CODE REVIEW

File: examples/raspicam/README.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+		```
+	+ create a buid dir and start compiling
+		```  
+		mkdir build
+		cd build
+		cmake ..
+		```
+	+  At this point you'll see something like 
+	```
+	-- CREATE OPENCV MODULE=1
```
Comment (ihhub) [inline examples/raspicam/README.md]: How does OpenCV relate to raspicam?

Comment (Captain-DarkO) [reply examples/raspicam/README.md]: I added the OpenCV support compilation step as people might use the raspicam library for other projects too.
Will remove that section.

Comment (Captain-DarkO) [reply examples/raspicam/README.md]: Done. Please review.

**Comment by ihhub** [comment]
Thank you very much for contributing to the project!

**Comment by Captain-DarkO** [comment]
Cheers! (:

**Pull request merged**